classes/Commands/UpdateCommand.php

```diff
@@ -0,0 +1,200 @@
+<?php
+
+/**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Academic Free License 3.0 (AFL-3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/AFL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/AFL-3.0 Academic Free License 3.0 (AFL-3.0)
+ */
+
+namespace PrestaShop\Module\AutoUpgrade\Commands;
+
+use Exception;
+use PrestaShop\Module\AutoUpgrade\ErrorHandler;
+use PrestaShop\Module\AutoUpgrade\Log\CliLogger;
+use PrestaShop\Module\AutoUpgrade\Log\Logger;
+use PrestaShop\Module\AutoUpgrade\Log\StreamedLogger;
+use PrestaShop\Module\AutoUpgrade\Task\ExitCode;
+use PrestaShop\Module\AutoUpgrade\Task\Miscellaneous\UpdateConfig;
+use PrestaShop\Module\AutoUpgrade\Task\Runner\AllUpgradeTasks;
+use PrestaShop\Module\AutoUpgrade\UpgradeContainer;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class UpdateCommand extends Command
+{
+    /**
+     * @var Logger
+     */
+    private $logger;
+
+    /**
+     * @var string
+     */
+    protected static $defaultName = 'update:start';
+
+    protected function configure(): void
+    {
+        $this
+            ->setDescription('Update your store.')
+            ->setHelp(
+                'This command allows you to start the update process. ' .
+                'Advanced users can refer to the https://devdocs.prestashop-project.org/8/basics/keeping-up-to-date/upgrade-module/upgrade-cli/ for further details on available actions'
```

Comment:
Suggestion: instead of having scattered links to devdocs all over the place, create a `DeveloperDocumentation` PHP class with constants for all these links, and keep all the links in the same place.

Also these links target the documentation for prestashop v8. Maybe we would need a separate doc for the module? so it does not depend on a core version.

I'm basically saying "let's create `https://devdocs.prestashop-project.org/upgrade/...`

Reply:
Good idea for the class with DOC constants. I use the v8 doc because the upgrade part is much more complete. We might have a module specific doc. To see to add it to the backlog.